client/App.vue

```diff
@@ -84,6 +90,9 @@ export default {
     onNotificationClose() {
       this.notification.show = false;
     },
+    onAnnouncementClose() {
```

Comment:
Shouldn't these be arrow functions?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions

Reply:
not sure if i understand the question correctly in this context

if vue.js you register the methods as part of the `methods` key. The value to `methods` is an object (dictionary) that contains the functions to register.
If we use arrow function here, then the syntax would still be very similar, except for containing few extra symbols. Something like:
`onAnnouncementClose: () => {...}`

Reply:
I misunderstood the context of the code. Shawn is obviously right and method definitions do not come with scoping issues and therefore arrow isn't applicable.